src/styles/searchresults.css

```diff
@@ -0,0 +1,25 @@
+.card-image {
+  border-radius: 20%;
+  width: 100%;
+  height: auto;
+  margin: auto;
+  padding: 30px;
+  object-fit: cover;
+}
+
+.card-image-parent {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+  grid-gap: 20px;
+  width: 95%
+}
+
+@media screen and (max-width: 767px) {
+  .app {
+    flex-direction: column;
+  }
```

Comment:
Its bad practice to modify classes like this that are not in the same file, if you want to add a media query to a class it should always be done in the same file as the class was originally declared